internal/lockclient/simple_client.go

```diff
@@ -16,12 +17,14 @@ var _ Config = (*lockservice.SimpleConfig)(nil)
 
 // SimpleClient implements Client, the lockclient for LocKey.
 type SimpleClient struct {
-	config lockservice.SimpleConfig
-	cache  cache.LRUCache
+	config *lockservice.SimpleConfig
+	cache  *cache.LRUCache
+	mu     sync.Mutex
 }
 
-// NewSimpleKey returns a new SimpleKey of the given value.
-func NewSimpleClient(config lockservice.SimpleConfig, cache cache.LRUCache) *SimpleClient {
+// NewSimpleClient returns a new SimpleKey of the given value.
```

Comment:
Returns a `SimpleKey`?? I think that needs to be changed

Reply:
Oops, I think someone copy pasted this.